src/core/utils/CommonHelpers.ts

```diff
@@ -47,4 +47,13 @@ export class CommonHelpers {
             await this.sleep(interval);
         }
     }
+    public static parseObjectProperties(obj, parse): void {
```

Comment:
The params `parse` type is `any`, I'm not sure it's a good idea.
my suggestion will be a method in helper class should return something we can predict.